c++/src/kj/compat/url.h

```diff
@@ -139,9 +151,15 @@ struct Url {
   static Maybe<Url> tryParse(StringPtr text, Context context = REMOTE_HREF, Options options = {});
   // Parse an absolute URL.
 
+  static Maybe<Url> tryParseWithoutContext(StringPtr text, Options options = {});
```

Comment:
Totally optional nit: what about just changing `tryParse` to take a `kj::Maybe<Context>`? Then to not pass in context you just pass in `nullptr`? I could see an argument either way but figured I'd mention it just as a discussion point that can be ignored.

Reply:
Hmmmmm, if you think that fits with the codebase style, that's probably a better choice for maintaining a smaller public API that we have to maintain. IIRC `Maybe` doesn't have an `explicit`-decorated ctor, so we should be able to do that without breaking people.